docs/UserGuide.md

```diff
@@ -227,6 +216,24 @@ Examples:
 * `addEvent n/Shoe Sale 30% d/30-05-2022 t/11:00 p/Discount on all shoes for up to 30%`
 * `addEvent n/Banana Discount 10% d/20-04-2022 t/14:00 p/10% discount on all bananas`
 
+### Locating events by event title: `findEvents`
+
+Finds events whose event titles contain any of the given keywords.
+
+Format: `findEvents KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g. `hans` will match `Hans`
+* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+* Only the event title is searched.
+* Only full words will be matched e.g. `Han` will not match `Hans`
+* Events matching at least one keyword will be returned (i.e. `OR` search).
+  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
```

Comment:
Perhaps would be better to change all the examples here to match the event titles. E.g 'car' instead of 'hans'